test/ng/directive/formSpec.js

```diff
@@ -460,6 +466,7 @@ describe('form', function() {
         .runs(function() {
           expect(doc.html()).toBe('');
           expect(destroyed).toBe(true);
+          expect(reloadPrevented).toBe('never called');
```

Comment:
So, it turns out the cause is again that Chrome 60+ on Windows does not fire `submit` events if the form is not attached to the DOM. Changed the expectation to verify that the `submit` event is eithr never fired or (if fired) the reload is prevented.